src/components/VencordSettings/UpdaterTab.tsx

```diff
@@ -224,29 +250,35 @@ function Updater() {
                 Get notified when an automatic update completes
             </Switch>
 
-            <Forms.FormTitle tag="h5">Repo</Forms.FormTitle>
-
-            <Forms.FormText className="vc-text-selectable">
-                {repoPending
-                    ? repo
-                    : err
-                        ? "Failed to retrieve - check console"
-                        : (
-                            <Link href={repo}>
-                                {repo.split("/").slice(-2).join("/")}
-                            </Link>
-                        )
-                }
-                {" "}(<HashLink hash={gitHash} repo={repo} disabled={repoPending} />)
-            </Forms.FormText>
+            {updaterContent}
```

Comment:
There doesn't seem to be any reason for this to require a variable, is there?
```suggestion
            <UpdaterContent />
```